api/v1alpha1/extendeddaemonset_types.go

```diff
@@ -69,6 +69,17 @@ type ExtendedDaemonSetSpecStrategyRollingUpdate struct {
 	SlowStartAdditiveIncrease *intstr.IntOrString `json:"slowStartAdditiveIncrease,omitempty"`
 }
 
+// ExtendedDaemonSetSpecStrategyCanaryValidationMode type representing the ExtendedDaemonSetSpecStrategyCanary validation mode.
+// +kubebuilder:validation:Enum=Auto;Manual
+type ExtendedDaemonSetSpecStrategyCanaryValidationMode string
+
+const (
+	// ExtendedDaemonSetSpecStrategyCanaryValidationModeAuto the ExtendedDaemonSetSpecStrategyCanary automatic validation mode.
+	ExtendedDaemonSetSpecStrategyCanaryValidationModeAuto ExtendedDaemonSetSpecStrategyCanaryValidationMode = "Auto"
+	// ExtendedDaemonSetSpecStrategyCanaryValidationModeManual the ExtendedDaemonSetSpecStrategyCanary manual validation mode.
+	ExtendedDaemonSetSpecStrategyCanaryValidationModeManual ExtendedDaemonSetSpecStrategyCanaryValidationMode = "Manual"
```

Comment:
other enums are starting by a lower case. could you update "Auto" and "Manual" to "auto" and "manual"